tests/integration/golang/aim/run/search_test.go

```diff
@@ -262,20 +263,33 @@ func (s *SearchTestSuite) SetupTest() {
 	s.Require().Nil(err)
 }
 
+func (s *SearchTestSuite) TestNoRequiredExperimentNames() {
```

Comment:
I think we append append `_Error` for the tests which expect error, eg `TestValidator_Error`